client/src/Components/Login.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import LoginModal from './LoginModal.jsx'
+import { Navbar } from 'react-bootstrap';
+import { NavDropdown } from 'react-bootstrap';
+import { Nav } from 'react-bootstrap';
+import { FormGroup } from 'react-bootstrap';
+import { FormControl } from 'react-bootstrap';
+import { NavItem } from 'react-bootstrap';
+import styles from './../styles/Login.css';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: ''
+    };
+    
+  }
+
+  handleSubmit(username) {
+    this.setState({ 
+      username: username
+    })
+    console.log(this.state)
+  }
+
+  render() {
+    if(this.state.username === '') {
```

Comment:
The two blocks in this `if else` statement are very similar. You should one of the blocks into its own component that can be modified slightly depending on whether the user is signed in (it could be passed as a prop). That would simplify this render function a lot.